### ISSUE TITLE
Not working on Intel Edison

### ISSUE THREAD

**Issue by brice-morin** [yeoman/yeoman#1462]
Hello,

It seems Yeoman is not working on Intel Edison. 

```
Error: EACCES, permission denied '/home/root/.config'
    at Object.fs.mkdirSync (fs.js:642:18)
    at sync (/usr/lib/node_modules/yo/node_modules/configstore/node_modules/mkdirp/index.js:70:13)
    at Function.sync (/usr/lib/node_modules/yo/node_modules/configstore/node_modules/mkdirp/index.js:76:24)
    at Object.create.all.get (/usr/lib/node_modules/yo/node_modules/configstore/index.js:41:13)
    at Object.Configstore (/usr/lib/node_modules/yo/node_modules/configstore/index.js:27:44)
    at new Insight (/usr/lib/node_modules/yo/node_modules/insight/lib/index.js:26:34)
    at Object.<anonymous> (/usr/lib/node_modules/yo/cli.js:123:15)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)

```

After creating a file `/home/root/.config` to help a little bit, I got:

```
Error: ENOTDIR, not a directory '/home/root/.config/configstore/insight-yo.yml'
    at Object.fs.openSync (evalmachine.<anonymous>:427:18)
    at Object.fs.readFileSync (evalmachine.<anonymous>:284:15)
    at Object.create.all.get (/usr/lib/node_modules/yo/node_modules/configstore/index.js:34:29)
    at Object.Configstore (/usr/lib/node_modules/yo/node_modules/configstore/index.js:27:44)
    at new Insight (/usr/lib/node_modules/yo/node_modules/insight/lib/index.js:26:34)
    at Object.<anonymous> (/usr/lib/node_modules/yo/cli.js:123:15)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
```

Maybe the problem comes from the generator I am using, but AFAIK, it seems quite Yeoman-specific.

Or maybe I am doing something wrong... but I could use yo on my PC before I tried on Intel Edison.

I am using an Intel Edison, with default OS (based on Linux Yocto, I think), and the pre-installed node.js/npm (v0.10.28)

**Comment by SBoudrias** [comment]
"permission denied" you have a permission issue... Google for chmod.

**Comment by brice-morin** [comment]
Fair enough... Though I installed and used Yeoman with the same user (I am currently using root only on the Edison as I am just playing with it, I am not too concerned with security at this point), so I would expect Yeoman to work out of the box in this condition (as it does on my standard PC). In this root-only context I am not quite sure what the permission problem is, and how I can solve it. And I did not find any evidence on Google that anyone has ever managed (or dare to try) Yeoman on Intel Edison/Yocto.
